x-pack/elastic-agent/spec/packetbeat.yml

```diff
@@ -0,0 +1,21 @@
+name: Packetbeat
+cmd: packetbeat
+args: ['-E', 'setup.ilm.enabled=false', '-E', 'setup.template.enabled=false', '-E', 'management.mode=x-pack-fleet', '-E', 'management.enabled=true', '-E', 'logging.level=debug']
+artifact: beats/packetbeat
+rules:
+  - filter_values:
```

Comment:
please include `- fix_stream: {}` which will populate dataset and namespace fields

Reply:
also think whether or not `inject_index` rule is needed, this rule adds processor injecting index to events in form of `type-dataset-namespace` (to follow indexing strategy) while `dataset` comes from stream node and `namespace` from `inputs` type is something which they can have in common e.g `logs` in filebeat.

Reply:
@michalpristas so, both the `fix_stream` and `inject_index` stuff is actually handled in packetbeat itself (see the [PR diff](https://github.com/elastic/beats/pull/22134/files#diff-463b4d620fc5649dde5c385800051ecfb1a6952d51c858c59fe5a43184fee3ccR54)) @ruflin was mentioning that we were trying to deprecate many of the transpiler rules, so I made this as minimal as possible and moved the logic into the go code.